### DISCUSSION TITLE
在阿里云linux服务器上安装alist服务并运行后，添加阿里云盘和115网盘，播放视频时候可以明显发现115速度很慢怎么办

### DISCUSSION THREAD

**Discussion by optimuszhao** [AlistGo/alist#4493] [Q&A]
作为对比，我尝试了在本地mac电脑也安装了alist，并添加了115 和阿里云盘，发现本地的服务就很快，这个是受到我服务器的带宽限制吗？那为什么服务器上的alist服务访问阿里云盘就很快呢？请问下大家，这个问题如何解决，因为我希望分享给我外地的朋友一起使用，所以希望alist服务运行在服务器上。

**Comment by xhofe** [comment]
- https://alist.nn.ci/zh/guide/drivers/aliyundrive_open.html#%E9%BB%98%E8%AE%A4%E4%BD%BF%E7%94%A8%E7%9A%84%E4%B8%8B%E8%BD%BD%E6%96%B9%E5%BC%8F
- https://alist.nn.ci/zh/guide/drivers/115.html#%E9%BB%98%E8%AE%A4%E4%BD%BF%E7%94%A8%E7%9A%84%E4%B8%8B%E8%BD%BD%E6%96%B9%E5%BC%8F
